frontend/src/components/widget/List/RawList.js

```diff
@@ -79,11 +79,17 @@ export class RawList0 extends PureComponent {
   }
 
   componentDidUpdate(prevProps) {
-    const { listHash } = this.props;
+    const { listHash, list = [] } = this.props;
     let changedValues = {};
 
-    if (listHash !== prevProps.listHash) {
-      const { list, mandatory, defaultValue, selected, emptyText } = this.props;
+    const { listHash: prevListHash, list: prevList = [] } = prevProps;
+
+    const loadDropdown =
+      (listHash !== prevListHash && (list.length > 0 || prevList.length > 0)) ||
+      (listHash && !prevListHash);
```

Comment:
pls add a js comment to explain the meaning/reason of those expressions.

Reply:
i mean, using that `listHash` thing it's a workaround anyways. I don't want to add more FUD on top of it.
So until we refactor it, we can at least document what's the meaning/which cases we want to solve with it.
So later on, when we will refactor, we will understand what shall we do.